.github/workflows/maven-publish.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish snapshots to maven
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'main'
+      - '1.*'
```

Comment:
1.x do not have this task. It started from 2.x. Would recommend to remove this to avoid failures.